src/test/scala/beam/utils/scenario/UrbanSimScenarioLoaderTest.scala

```diff
@@ -90,11 +98,45 @@ class UrbanSimScenarioLoaderTest extends AsyncWordSpec with Matchers with Before
 
       assertCarNumbers(res, "1" -> 2, "2" -> 2, "3" -> 2)
     }
+
+  }
+
+  "UrbanSimScenarioLoader with vehicleAdjustmentMethod = STATIC_FROM_FILE "should {
+    "assign vehicles properly in case1" in {
+
+      val matsimConfig = new MatSimBeamConfigBuilder(staticVehiclesConfig).buildMatSimConf()
+      val emptyScenario = ScenarioBuilder(matsimConfig, beamScenario.network).build
+      val staticVehicleBeamConfig = BeamConfig(staticVehiclesConfig)
+      val staticGeoUtils = new GeoUtilsImpl(staticVehicleBeamConfig);
+      when(beamScenario.beamConfig).thenReturn(staticVehicleBeamConfig);
+      lazy val staticBeamScenario: BeamScenario = loadScenario(staticVehicleBeamConfig);
+      val staticScenarioSource = buildUrbansimV2ScenarioSource(staticGeoUtils, staticVehicleBeamConfig);
+      val urbanSimScenario = new UrbanSimScenarioLoader(emptyScenario, staticBeamScenario, staticScenarioSource, geoUtils)
+      urbanSimScenario.loadScenario()
+
+      val vehicleIds = emptyScenario.getHouseholds.getHouseholds.get(Id.create(2, classOf[Household])).getVehicleIds
+      val vehicleMap = staticBeamScenario.privateVehicles.toMap
+      val vehiclesBelongToSpecificHousehold = vehicleIds.map(x => Id.create(x, classOf[BeamVehicle])).flatMap(x => vehicleMap get x);
+      vehiclesBelongToSpecificHousehold.count(x => x.beamVehicleType.id.compareTo(
+        Id.create("Bicycle", classOf[BeamVehicleType])
+      ) == 0) shouldBe 7
+    }
+
   }
 
   override protected def beforeEach(): Unit =
     idIter = Iterator.from(1)
 
+  def staticVehiclesConfig: com.typesafe.config.Config =
+    ConfigFactory
+      .parseString(s"""
+                      |beam.agentsim.simulationName = "beamville-urbansimv2_static"
+                      |beam.agentsim.agents.vehicles.vehiclesFilePath = $${beam.inputDirectory}"/vehicles-test.csv"
+                      |beam.agentsim.agents.vehicles.vehicleAdjustmentMethod = "STATIC_FROM_FILE"
+                    """.stripMargin)
+      .withFallback(testConfig("test/input/beamville/beam-urbansimv2.conf"))
+      .resolve()
```

Comment:
let's put this config initialization inside the test, no need to create external variable for a test